content/en/blog/2023/end-user-discussions-04.md

```diff
@@ -0,0 +1,111 @@
+---
+title: OpenTelemetry End-User Discussions Summary for April 2023
+linkTitle: End-User Discussions April 2023
+date: 2023-05-02
+author: '[Pranay Prateek](https://github.com/pranay01) (SigNoz)'
+---
+
+For the month of April 2023, the OpenTelemetry end-user group meet took place
+for users in the Asia-Pacific (APAC) region.
+
+The discussions take place using a
+[Lean Coffee format](https://agilecoffee.com/leancoffee/), whereby folks are
+invited to post their topics to the
+[Agile Coffee board like this one](http://agile.coffee/#2f83c1c1-918c-4c78-8671-194b2e9d8e54),
+and everyone in attendance votes on what they want to talk about.
+
+## What we talked about
+
+We talked about evangelizing the adoption of OpenTelemetry in a big organization
+and also discussed how to optimize observability data at scale.
+
+## Discussion Highlights
+
+Below is the summary of this month's discussion.
```

Comment:
```suggestion
Below is the summary of this month's discussion in a Q&A format. Questions were posed by end-users, and answers were provided by a mix of OTel community members and end-users.
```

Adding on the last bit to address @svrnm's comment. Hope this helps clarify things a bit.

Reply:
It clarifies yes, additionally I would like this to be more explicit about not being official: 

> The answers provided are opinions by the discussion participants and are given in the context of the discussion.

Reply:
How about we include a note at the beginning of each OTel EUG blog? @svrnm @avillela 

`Note:: The answers are provided by a mix of OTel community members and end-users to the best of their knowledge. The answers are not official recommendations by OpenTelemetry.`